dotcom-rendering/src/components/CartoonComponent.tsx

```diff
@@ -33,75 +66,18 @@ export const CartoonComponent = ({ format, element, switches }: Props) => {
 				}
 			`}
 		>
-			<Hide until="desktop">
-				{largeVariant?.images.map((image) => {
-					return (
-						<>
-							<Picture
-								master={image.url}
-								role={element.role}
-								format={{
-									display: format.display,
-									design: format.design,
-									theme: format.theme,
-								}}
-								alt={`${
-									element.alt ? `${element.alt}, ` : ''
-								}panel ${image.index + 1}`}
-								height={parseInt(image.fields.height, 10)}
-								width={parseInt(image.fields.width, 10)}
-								key={image.index}
-							/>
-							{switches?.lightbox === true &&
-								isWideEnough(image) &&
-								element.position !== undefined && (
-									<LightboxLink
-										role={element.role}
-										format={format}
-										elementId={element.elementId}
-										isMainMedia={true}
-										position={element.position}
-									/>
-								)}
-						</>
-					);
-				})}
-			</Hide>
-			<Hide from="desktop">
-				{smallVariant?.images.map((image) => {
-					return (
-						<>
-							<Picture
-								master={image.url}
-								role={element.role}
-								format={{
-									display: format.display,
-									design: format.design,
-									theme: format.theme,
-								}}
-								alt={`${
-									element.alt ? `${element.alt}, ` : ''
-								}panel ${image.index + 1}`}
-								height={parseInt(image.fields.height, 10)}
-								width={parseInt(image.fields.width, 10)}
-								key={image.index}
-							/>
-							{switches?.lightbox === true &&
-								isWideEnough(image) &&
-								element.position !== undefined && (
-									<LightboxLink
-										role={element.role}
-										format={format}
-										elementId={element.elementId}
-										isMainMedia={true}
-										position={element.position}
-									/>
-								)}
-						</>
-					);
-				})}
-			</Hide>
-
+			{smallVariant ? (
+				<>
+					<Hide until="desktop">
+						{largeVariant?.images.map(render)}
+					</Hide>
+					<Hide from="desktop">
+						{smallVariant?.images.map(render)}
+					</Hide>
```

Comment:
We tend to design mobile-first, so maybe switching these around would be better? Again, aware that this is simply moving code around.

Reply:
addressed here: https://github.com/guardian/dotcom-rendering/pull/9460/commits/314f619868672e9ef0641d1e021028be81c3cad6